executor/runtime/types/types.go

```diff
@@ -28,6 +28,7 @@ const (
 	KvmEnabledParam              = "titusParameter.agent.kvmEnabled"
 	assignIPv6AddressParam       = "titusParameter.agent.assignIPv6Address"
 	serviceMeshEnabledParam      = "titusParameter.agent.service.serviceMesh.enabled"
+	serviceMeshContainerParam    = "titusParameter.agent.service.serviceMesh.container"
```

Comment:
Only other minor nit: might want to make this `containerImage`, or `image`, so it's a bit more in your face what this property is for.